src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -655,6 +657,7 @@ const TimeEntryForm = props => {
             <FormGroup>
               <Label for="notes">Notes</Label>
               <Editor
+                disabled={!(canEditTimeEntry || canEditTimeEntryDescription) }
```

Comment:
```suggestion
                disabled={!canEditTimeEntryDescription}
```
`editTimeEntry` should be replaced by more granular permissions (`editTimeEntryDescription` in this case)